src/commands/disconnect.rs

```diff
@@ -0,0 +1,34 @@
+use std::sync::Arc;
+
+use anyhow::Result;
+use serenity::async_trait;
+use serenity::builder::CreateApplicationCommand;
+
+use crate::command_context::CommandContext;
+
+use super::{JoovyCommand, JoovyCommands};
+
+#[derive(Default, Debug)]
+pub struct Disconnect;
+
+#[async_trait]
+impl JoovyCommand for Disconnect {
+    fn create_application_command<'a>(
+        &self,
+        command: &'a mut CreateApplicationCommand,
+    ) -> &'a mut CreateApplicationCommand {
+        command
+            .name(JoovyCommands::Disconnect(Disconnect).as_ref())
+            .description("Disconnects from the voice channel")
```

Comment:
"Disconnects the bot from its current voice channel"

Reply:
Fixat!